common/lib/xmodule/xmodule/raw_module.py

```diff
@@ -22,7 +25,13 @@ class RawMixin(object):
 
     @classmethod
     def definition_from_xml(cls, xml_object, system):
-        return {'data': etree.tostring(xml_object, pretty_print=True, encoding='unicode')}, []
+        pre_tag_data = [etree.tostring(pre_tag_info) for pre_tag_info in xml_object.findall('pre')]
+        pre_tag_pattern = re.compile(PRE_TAG_REGEX)
```

Comment:
actually `.compile` will take time every time compiling the same regex; even though it would cause significant time difference, but compiling it once makes sense. 
Thoughts @DawoudSheraz ?

Reply:
Not sure if making it a class variable would sense as it is used in one function only. If we can get the time comparison of this function in the current state and one with regex being class variable, then we can make the best decision.

Reply:
Not a class variable actually. Just update the L#15 to this to avoid compiling it over and over for every unit in the course xml. 
`PRE_TAG_REGEX = re.compile(r'<pre>[\s\S]*?</pre>')`